janusgraph-server/src/test/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerClientTest.java

```diff
@@ -62,51 +65,127 @@ public void testGetVertexLabelByNameInvalidArgumentNullContext() {
         assertThrows(NullPointerException.class, () -> schemaManagerClient.getVertexLabelByName("test"));
     }
 
-    @Test
-    public void testGetVertexLabelByNameVertexLabelExists() {
-        final String vertexLabelName = "test";
+    @ParameterizedTest
```

Comment:
I suspect you need to add the following dependencies into pom.xml so that maven can execute it:
```xml
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-params</artifactId>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-engine</artifactId>
            <scope>test</scope>
        </dependency>
```

I had been wondering why my code coverage was so low in #2556 until I added these dependencies to janusgraph-cql/pom.xml. It seems otherwise, maven would not execute these parameterized tests.

Reply:
I just checked all tests are executed.

Reply:
You are right... now I am wondering why I needed them in janusgraph-cql/pom.xml...